libnetdata/inlined.h

```diff
@@ -42,6 +65,21 @@ static inline uint32_t simple_uhash(const char *name) {
     return hval;
 }
 
+static inline uint32_t simple_nuhash(const char *name,size_t len) {
```

Comment:
Code style, you should insert a space after a comma (`name, size_t len`), there are a lot of places in the code you could do that, I'm not going to write a comment for each because I will pollute the review log but you should do a cleanup.